### PR TITLE
feat: add http requests metrics doc

### PR DESCRIPTION
**Pull request by eddycharly**
## Related issue #

This PR adds http requests metrics doc.

### CODE REVIEW

File: content/en/docs/Monitoring/http-requests.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
+---
+title: Http Requests Counts
```
Comment (chipzoller) [inline content/en/docs/Monitoring/http-requests.md:2]: HTTP Request Count

**Review by chipzoller — CHANGES_REQUESTED** [review]
"HTTP" is all caps.

Which controllers emit this?

File: content/en/docs/Monitoring/http-requests.md (hunk 1 above)
Comment (eddycharly) [reply content/en/docs/Monitoring/http-requests.md]: Fixed

**Comment by chipzoller** [comment]
> > Which controllers emit this?
> 
> Is this really important ?

I honestly don't know. What do you think? Is it relevant or not?

**Comment by eddycharly** [comment]
> I honestly don't know. What do you think? Is it relevant or not?

I don't think it will be particularly useful.

**Comment by chipzoller** [comment]
Ok, then forget it.

**Review by chipzoller — APPROVED** [review]

File: content/en/docs/Monitoring/http-requests.md (hunk 1 above)
Comment (chipzoller) [reply content/en/docs/Monitoring/http-requests.md]: "Count" is one of those words which implies plural. It's one counter with numbers that can be greater than zero.

**Pull request merged**